test/kitchen/site-cookbooks/dd-agent-5/templates/datadog.conf.erb

```diff
@@ -0,0 +1,251 @@
+# Generated by Chef, local modifications will be overwritten
```

Comment:
do we need to include this complex template?

If I understand correctly, for the purposes of installing Agent 5 here, this template could only have `dd_url` and `api_key`, and we could also remove most of the defaults that are set in `dd-agent-5/attributes/default.rb` (I don't understand why we set all of these defaults: https://github.com/DataDog/datadog-agent/blob/78ceef29d369ff75f88bf97af763c0b60aed0493/test/kitchen/site-cookbooks/dd-agent-5/attributes/default.rb#L42-L87)

(seeing if we can keep things as simple as possible :) )

Reply:
True, I took it from the `chef-datadog` repo directly without thinking much, but we don't need most of what's there since we only create testing VMs.